src/main/java/frc/robot/util/Conversions.java

```diff
@@ -113,4 +117,20 @@ public static double ElevatorHeightToExtension(double height) {
     public static double ElevatorExtensionToHeight(double extension) {
       return (extension * Math.sin(ElevatorConstants.kElevatorAngle)) + ElevatorConstants.kElevatorBaseHeight;
     }
+
+    /**
+     * Converts between Absolute coordinate system pose of origin right of blue driver station
+     * to coordinate system where origin the current alliance driver station
+     * The transformation is reversible so there is no second function to convert back
```

Comment:
I think the proper name for this is "self-inverse" Reversible means that there exists a reverse transformation, here you mean that reverse transformation is identical to the original one.